features/features.go

```diff
@@ -465,14 +433,6 @@ var (
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
-	FeatureGateTranslateStreamCloseWebsocketRequests = newFeatureGate("TranslateStreamCloseWebsocketRequests").
-								reportProblemsToJiraComponent("kube-apiserver").
-								contactPerson("akashem").
-								productScope(kubernetes).
-								enhancementPR("https://github.com/kubernetes/enhancements/issues/4006").
-								enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-								mustRegister()
-
```

Comment:
Enabled by default in 1.30: https://github.com/kubernetes/kubernetes/blob/7cd7fea30d054efe4f2cc60073a29d2c618258ec/pkg/features/kube_features.go#L1823-L1826

@tkashem do you know if we should drop it?

Reply:
We'll keep it for now until these features are GA'ed in upstream.

Reply:
Unless it's locked on by default upstream, we should keep this until we GA the feature